src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -1,244 +1,417 @@
-should feel free to engage other developers with help. But primary should first
-enlist the backup. When developer expertise other than backup is needed, primary
-should arrange mutually agreeable times to engage those other developers.
-
-There shall be no expectation that a developer serving as primary can get any
-other work done beyond their customer response obligations. In slow weeks, its
-conceivable they can. But, there can be no implied assumption or expectation
-that this will be the case. Furthermore, even in a slow week, the primary may
-likely be able to find plenty of *low-hanging-fruit-type* tasks in the main
-repository issues to tackle and fix.
-
-During their communication's with the customer, the primary should avoid
-reply-all-type communications.
-
-For applicable incidents, the primary should prepare a set of bullets to
-form an outline of a *post-mortem* for the incident to be reviewed at VisIt_
-project meetings.
-
-The Backup's Role
------------------
+  * Developing a reproducer for developers.
+
+    * This may include any relevant user data files as well as approval, where
+      appropriate for world read access to such data.
 
-Ideally, like the rest of the development team, the backup is never called into
-action for customer response. However, there are cases where the backup may be
-needed. For this reason, the backup is asked to at least track and maintain
-knowledge and awareness of the issues the primary is supporting. A good way to
-achieve this is to selectively *watch* the visit-live-incidents repository.
-In cases where primary needs help in resolving an incident, the backup
-shall serve as the first line of defense. In addition, the backup should be
-ready to step up to the primary role in the (hopefully rare) occurrence that the
-primary is unable to maintain the one-hour response time goal. To the extent
-possible, the transition from backup to primary should be managed formally.
-Otherwise, the backup should simply listen in but otherwise not engage.
-
-Other Developer's Role
-----------------------
-
-In the ideal, the primary is able to handle all customer response activity and
-no other developers are interrupted. Thus, other developers should feel free to
-ignore customer inquiries and/or redirect customers who may contact them directly.
-One option to reduce visit-user traffic is to switch to receiving a daily digest.
-Other options might be email client side filters.
+  * Identifying a *low-hanging fruit* type issue and subsequently engaging
+    in the *housekeeping* work to resolve it.
+  * Determining if the user's issue is known (e.g. an issue ticket already exists).
+  * Updating a known issue with new information from this user, perhaps
+    adjusting labels on the issue or putting the issue back into the
+    UN-reviewed state for further discussion at a VisIt_ project meeting.
+  * Identifying and filing a new *product development* type issue ticket.
+
```

Comment:
How does *product development* type issue differ from all the other issues in our tracker?

Reply:
To follow up here, @markcmiller86 are you using *product development* as a type of *issue* or as the "name" of the issue-tracker in our main repo?

Reply:
> from @biagas: How does _product development_ type issue differ from all the other issues in our tracker?

> From @rusu24edward: To follow up here, @markcmiller86 are you using _product development_ as a type of _issue_ or as the "name" of the issue-tracker in our main repo?

Yeah, sorry for confusion here. I see two broad issue *categories*: Those associated with SRE activity (there is a user awaiting an answer, resolution might involve some diagonsing and developing a work-around, we might need a reproducer, etc.) and those associated with our main *product development* work (scheduled for a given release, assigned to a developer, some idea of time to complete, etc.)

SRE issues should go into the SRE issue tracker (which I probably misnamed as live-customer-support) and Product Development issues go into the visit-dav repo's issue tracker.

Does this make sense? I will try to make clear in the document.